### PR TITLE
[keytar] correct return types for signatures

### PR DESCRIPTION
**Pull request by shiftkey**
- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `tsc` without errors.
- ~~[ ] Run `npm run lint package-name` if a `tslint.json` is present.~~

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
   - [`getPassword`](https://github.com/atom/node-keytar/blob/master/src/main.cc#L27) - may return null if it can't find the account
   - [`findPassword`](https://github.com/atom/node-keytar/blob/master/src/main.cc#L47) - may return null if it can't find the account
   - [`deletePassword`](https://github.com/atom/node-keytar/blob/master/src/main.cc#L35) - returns a boolean to indicate success/failure
- [x] Increase the version number in the header if appropriate.
- ~~[ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "../tslint.json" }`.~~

### CODE REVIEW

File: keytar/index.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for keytar 3.0.0
+// Type definitions for keytar 3.0.2
 // Project: http://atom.github.io/node-keytar/
 // Definitions by: Milan Burda <https://github.com/miniak/>
```
Comment (anubmat) [inline keytar/index.d.ts]: @shiftkey , you need to add yourself to the contributors here..

File: keytar/index.d.ts [hunk 2]
```diff
@@ -56,4 +56,4 @@ export declare function replacePassword(service: string, account: string, passwo
  *
  * @returns the string password or null on failures.
```
Comment (anubmat) [inline keytar/index.d.ts]: this needs change too.

File: keytar/index.d.ts [hunk 3]
```diff
@@ -31,9 +31,9 @@ export declare function addPassword(service: string, account: string, password:
  * @param service The string service name.
  * @param account The string account name.
  *
- * @returns the string password or null on failures.
+ * @returns true on success, false on failure
  */
-export declare function deletePassword(service: string, account: string): string;
+export declare function deletePassword(service: string, account: string): boolean;
```
Comment (anubmat) [inline keytar/index.d.ts]: Can you add a test for this please..

File: keytar/index.d.ts (hunk 2 above)
Comment (shiftkey) [inline keytar/index.d.ts]: Not sure I understand - this method seems correct: https://github.com/atom/node-keytar/blob/master/src/main.cc#L38

File: keytar/index.d.ts (hunk 3 above)
Comment (shiftkey) [inline keytar/index.d.ts]: I took a shot at this in 1753b8c0c to validate the return values for all these methods. Let me know how that feels.

**Pull request merged**